docs/manual/en/introduction/Installation.html

```diff
@@ -10,149 +10,245 @@
 		<h1>[name]</h1>
 
 		<p>
-			You can install three.js with [link:https://www.npmjs.com/ npm] and modern build tools, or get started quickly with just static hosting or a CDN. For most users, installing from npm is the best choice.
+			Most web developers install JavaScript libraries like three.js from the [link:https://www.npmjs.com/ npm package registry], then develop their project using [link:https://eloquentjavascript.net/10_modules.html#h_zWTXAU93DC build tools] like Vite or Webpack. Because three.js is also popular with many communities that may not be as familiar with these tools, this guide also describes a simplified "static" installation using a [link:https://developer.mozilla.org/en-US/docs/Glossary/CDN CDN] instead of a build tool.
```

Comment:
What is the `#h_zWTXAU93DC` part of the link? It doesn't work without it?

Reply:
Links to the section header, *Building and Bundling*, not sure why it isn't a human-readable anchor. 🤷‍♂️